xtask/src/build.rs

```diff
@@ -9,27 +10,35 @@ pub fn build(params: BuildParams) -> anyhow::Result<()> {
         return Ok(());
     }
 
+    // Erase the log file if it already exists
+    std::fs::File::create(&params.log_path)?.set_len(0)?;
```

Comment:
1. totally right, both are not needed, I will remove `set_len`
2. Nice edge case, I didn't think about it yesterday. If I'm correct there is no option in the File struct to create all the but needs to be done in two step with `std::fs::create_dir_all`

Reply:
1. Totally right, both are not needed. I will remove `set_len`
2. Nice edge case. I didn't think about it yesterday. If I'm correct, there is no option in the `File` struct to create all the directories, but it needs to be done in two steps with `std::fs::create_dir_all`
3. Probably a fallback into the default log position firstly, and if that fails too, maybe just print stuff to stdout? It is absolutely doable!

Reply:
Ok, if the creation of the folder specified in the path or the file itself is a problem, then stdout is the fallback. Implementing this was a little more complex than expected, but I hope to have come up with a versatile and good enough solution

Reply:
FWIW, I was more thinking of ignoring the failure and discarding output (although maybe with a warning). This is not critical data, you can always re-run the code.

Reply:
Oh, okay, then I added a small modification. However, for how the duct is built, even discarding the output would require calling a method on the expression. It would be something like `expression.stdout_null().stderr_null().run()` instead of `expression.stderr_to_stdout().stdout_file(log_file).run()`. 

If you prefer, it would be really easy to switch to discarding the output instead of printing it to stdout in the current implementation, so let me know which one you think is more appropriate!

Reply:
I guess it's fine for now